packages/clerk-js/src/ui/components/OrganizationProfile/ProfileSettingsPage.tsx

```diff
@@ -44,6 +44,14 @@ export const ProfileSettingsPage = withCardStateProvider(() => {
   const uploadAvatar = (file: File) => {
     return organization.setLogo({ file }).then(() => {
       setAvatarChanged(true);
+      card.setIdle();
+    });
```

Comment:
Looks like we're missing error handling here (thats on me), can we add a simple 
```
      .catch(err => handleError(err, [], card.setError));
``` 
here and on `onAvatarRemove`?